core/domain/exp_domain.py

```diff
@@ -1249,8 +1249,8 @@ def __init__(
         """Initializes a State domain object.
 
         Args:
-            content: list(Content). The contents displayed to the reader in
-                this state. This list must have only one element.
+            content: list(SubtitledHtml). The contents displayed to the reader
```

Comment:
@vojtechjelinek, will this be a list? (not sure!)

Reply:
Yes, you're right! Done.